gateway/webhook/webhookTransformer.go

```diff
@@ -13,10 +13,13 @@ import (
 	"github.com/rudderlabs/rudder-server/utils/misc"
 )
 
+// transformerResponseT will be populated using JSON unmarshall
+// so we need to make fields public
 type transformerResponseT struct {
-	output     []byte
-	err        string
-	statusCode int
+	Output         map[string]interface{} `json:"output"`
+	Err            string                 `json:"error"`
+	StatusCode     int                    `json:"statusCode"`
+	OutputToSource map[string]interface{} `json:"outputToSource"`
```

Comment:
Are we confident that we will never have to return anything else other than a json object in our response? If not, this should be converted to a `[]byte`

Reply:
Done